libbeat/idxmgmt/ilm/ilm_test.go

```diff
@@ -74,6 +74,49 @@ func TestDefaultSupport_Init(t *testing.T) {
 		assert.Equal(Alias{Name: "alias", Pattern: "01"}, s.Alias())
 	})
 
+	t.Run("with custom alias config with fieldref", func(t *testing.T) {
+		tmp, err := DefaultSupport(nil, info, common.MustNewConfigFrom(
+			map[string]interface{}{
+				"enabled":        true,
+				"name":           "test-%{[agent.version]}",
```

Comment:
`name` is not part of the config and not used, the default `alias.name` is defined by the `beat.Info` passed in. I suggest to remove `name` as it is confusing. Also in the second example below.